Packs/AWSCompromisedEC2Response/Playbooks/playbook-AWS_Compromised_EC2_Response.yml

```diff
@@ -0,0 +1,618 @@
+      region: {}
+      roleArn: {}
+      roleSessionDuration: {}
+      roleSessionName: {}
+      waiterDelay: {}
+      waiterMaxAttempts: {}
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: AWS - EC2
+      description: A waiter function that runs every 15  seconds until a successful
+        state is reached.
+      id: 4d78360c-4262-4a90-8d5c-fd30993cfea6
+      iscommand: true
+      name: Wait for Forensic Instance Initialization
+      script: AWS - EC2|||aws-ec2-waiter-instance-running
+      type: regular
+      version: -1
+    taskid: 4d78360c-4262-4a90-8d5c-fd30993cfea6
+    timertriggers: []
+    type: regular
+    view: |-
+      {
+        "position": {
+          "x": -230,
+          "y": 1245
+        }
+      }
+  "15":
+    id: "15"
+    ignoreworker: false
+    nexttasks:
+      '#none#':
+      - "16"
+    note: false
+    quietmode: 0
+    scriptarguments:
+      CidrBlock:
+        complex:
+          accessor: ipAddressV4
+          root: incident.service.action.networkConnectionAction.remoteIpDetails
+          transformers:
+          - args:
+              prefix: {}
+              suffix:
+                value:
+                  simple: /32
+            operator: concat
+      Code: {}
+      DryRun: {}
+      Egress:
+        simple: "True"
+      From:
+        simple: "0"
+      Ipv6CidrBlock: {}
+      NetworkAclId:
+        simple: acl-0780ee09f0dbd6f7c
+      Protocol:
+        simple: "-1"
+      RuleAction:
+        simple: deny
+      RuleNumber:
+        simple: "1"
+      To:
+        simple: "65535"
+      Type: {}
+      region: {}
+      roleArn: {}
+      roleSessionDuration: {}
+      roleSessionName: {}
+    separatecontext: false
+    skipunavailable: false
+    task:
+      brand: AWS - EC2
+      description: Creates an entry (a rule) in a network ACL with the specified rule
+        number.
+      id: 4e646bda-4203-4c68-8118-9195f1e66be0
+      iscommand: true
+      name: Create Block ACL Entry for Offending IP (Outbound)
+      script: AWS - EC2|||aws-ec2-create-network-acl-entry
+      type: regular
+      version: -1
+    taskid: 4e646bda-4203-4c68-8118-9195f1e66be0
+    timertriggers: []
+    type: regular
+    view: |-
+      {
+        "position": {
+          "x": 300,
+          "y": 720
+        }
+      }
+  "16":
+    id: "16"
+    ignoreworker: false
+    note: false
+    quietmode: 0
+    scriptarguments:
+      CidrBlock:
+        complex:
+          accessor: ipAddressV4
+          root: incident.service.action.networkConnectionAction.remoteIpDetails
+          transformers:
+          - args:
+              prefix: {}
+              suffix:
+                value:
+                  simple: /32
+            operator: concat
+      Code: {}
+      DryRun: {}
+      Egress:
+        simple: "False"
+      From:
+        simple: "0"
+      Ipv6CidrBlock: {}
+      NetworkAclId:
+        simple: acl-0780ee09f0dbd6f7c
```

Comment:
Please replacer the hard-coded `NetworkAclId` input with playbook input / some other non hard-coded input.